services/frontend/src/modules/menteeList/MenteeList.tsx

```diff
@@ -0,0 +1,80 @@
+// import { ExternalLinkIcon } from '@chakra-ui/icons'
+import {
+  Box,
+  Container,
+  Heading,
+  Link,
+  Spinner,
+  VStack,
+} from '@chakra-ui/react'
+import React from 'react'
+import { auth } from '../../clients/firebase'
+import { useAuthState } from 'react-firebase-hooks/auth'
+import { useGetMenteeQuery } from '../../generates/graphql'
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+
+export const MenteeList: React.FC = () => {
+  const [user, _loading] = useAuthState(auth)
+  if (!user) {
+    return null
+  }
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const { data, loading } = useGetMenteeQuery({
+    variables: { userId: user.uid || '' },
+  })
```

Comment:
React の hooks の制限に 「フックを呼び出すのはトップレベルのみ」というものがあるので、以下のように修正してみてください！
https://ja.reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

```suggestion
  const [user, _loading] = useAuthState(auth)

  // eslint-disable-next-line react-hooks/rules-of-hooks
  const { data, loading } = useGetMenteeQuery({
    variables: { userId: user.uid || '' },
    skip: !user
  })
  
  if (!user) {
    return null
  }
```

Reply:
めちゃくちゃ基本的なとこでした笑
ありがとうございます